### PR TITLE
Token icon and name in deatils.

### PR DESCRIPTION
**Pull request by mg6maciej**

### CODE REVIEW

**Comment by jakubsta** [comment]
LGTM

File: packages/widgets/src/components/TokenLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  ['ethereum:0xe41d2489571d322189246dafa5ebde1f4699f498', zerox],
+  ['ethereum:0x744d70fdbe2ba4cf95131626614a1763df805b9e', status],
+]);
+
+interface ITokenLogoProps {
+  className: any;
+  asset: string;
+}
+
+class TokenLogo extends Component<ITokenLogoProps, {}> {
```
Comment (jakubsta) [inline packages/widgets/src/components/TokenLogo.tsx]: Could you rewrite it to PureComponent?

File: packages/widgets/src/components/TokenName.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { WIDGET_NETWORKS } from '@userfeeds/apps-components/src/Form/Asset';
+
+interface ITokenNameProps {
+  asset: string;
+}
+
+class TokenName extends Component<ITokenNameProps, {}> {
```
Comment (jakubsta) [inline packages/widgets/src/components/TokenName.tsx]: Same as above

File: packages/widgets/src/components/TokenLogo.tsx (hunk 1 above)
Comment (mg6maciej) [reply packages/widgets/src/components/TokenLogo.tsx]: Updated.

File: packages/widgets/src/components/TokenName.tsx (hunk 2 above)
Comment (mg6maciej) [reply packages/widgets/src/components/TokenName.tsx]: Same as above.

**Pull request merged**